Mathlib/Data/Int/Parity.lean

```diff
@@ -73,79 +74,85 @@ theorem even_or_odd (n : ℤ) : Even n ∨ Odd n :=
   Or.imp_right odd_iff_not_even.2 <| em <| Even n
 #align int.even_or_odd Int.even_or_odd
 
+-- Porting note: TODO. I'm struggling that `Even` uses `k + k` while Odd uses `2 * k`,
+-- does this proof need golfing?
+-- mathlib3 port: `simpa only [← two_mul, exists_or, ← Odd, ← Even] using even_or_odd n`
```

Comment:
I think it's fine to keep both this comment and your proof. In the future, someone can revisit.

Reply:
sounds good